### PR TITLE
minor text edits

### PR DESCRIPTION
**Pull request by lambdanon**

### CODE REVIEW

**Review by q-horton — CHANGES_REQUESTED** [review]
BE/ME change is good. Prerequisites have come directly from the source, and just lay out expected knowledge, so it's not great to suggest that it's only needed if you're in a specific specialisation. Perhaps just swap to a recommended prerequisite if you're trying to shift focus to those in the required course list.

**Comment by lambdanon** [comment]
hmm I guess i'll just have another small section for recommended prerequisites. ill have a look at the other courses if they also have recommended prereqs and fill those in as well.

File: 2023/chapters/Electives.tex [hunk 1]
```diff
@@ -12,7 +12,7 @@ \section{Electrical Engineering}
         \item \hyperlink{COMP4702}{COMP4702}
         \item \hyperlink{METR4202}{METR4202}
         \item \hyperlink{METR6203}{METR6203}
-        \item[]
+        \item \hyperlink{CCSE4011}{CSSE4011}
```
Comment (q-horton) [inline 2023/chapters/Electives.tex:15]: CCSE -> CSSE

File: 2023/chapters/Electives.tex [hunk 2]
```diff
@@ -55,6 +55,6 @@ \section{Computer Science}
         \item \hyperlink{COMP3702}{COMP3702}
         \item \hyperlink{COMP3710}{COMP3710}
         \item \hyperlink{COMP4702}{COMP4702}
-        \item[]
+        \item \hyperlink{CSSE3010}{CSSE3010}
```
Comment (q-horton) [inline 2023/chapters/Electives.tex:58]: This is the first item in the list

**Review by q-horton — CHANGES_REQUESTED** [review]
Just a couple small things, otherwise lgtm

File: 2023/chapters/Electives.tex (hunk 1 above)
Comment (lambdanon) [reply 2023/chapters/Electives.tex:15]: oh i swear i fixed that lmao

File: 2023/chapters/Electives.tex (hunk 2 above)
Comment (lambdanon) [reply 2023/chapters/Electives.tex:58]: nah its not, thats for software engg. ill fix up the other mistake

Comment (q-horton) [reply 2023/chapters/Electives.tex:58]: Ah yea, whoops, didn't notice the compression in the middle lol

**Pull request merged**